client/src/i18n/en/errors.json

```diff
@@ -25,6 +25,7 @@
     "ER_BAD_FIELD_ERROR" : "Column does not exist in database.",
     "ER_ROW_IS_REFERENCED" : "Cannot delete entity because entity is used in another table.",
     "ER_ROW_IS_REFERENCED_2" : "Cannot delete entity because entity is used in another table.",
+    "ER_PREVENT_NEGATIVE_QUANTITY_IN_STOCK" : "There are stock movements, which may overconsume the quantity in stock and generate negative quantity in stock",
```

Comment:
I see that this error message will be used in multiple contexts.  Ideally there would be a separate message for each place this issue can come up.  Eg, one for stock entry that starts with:   "The proposed stock entry may overconsume..."
and for stock adjustment:  "The proposed stock adjustment may overconsume..."
Also, if you feel it is appropriate, I suggest replacing "may" with "will".

Whether you decided to add more error messages or not, please remove the comma.
"There are stock movements which may..."

Reply:
```` Also, what is the strategy for dealing with sites that already have negative quantities? ````
In this case alone the positive adjustment is required, I just tested it with the database of IMCK and it works well

Reply:
```` Also, what is the strategy for dealing with sites that already have negative quantities? ````
In this case alone the positive adjustment is required, I just tested it with the database of IMCK and it works well

Reply:
Once the more detailed error message is display, I agree that we don't need more detail in the temporary warning at the top of the page.